unity/XRTProject/Assets/Scripts/Networking/ApiRequest.cs

```diff
@@ -61,7 +61,7 @@ public ApiRequest HandleResponse(Action<object> responsePayload)
 
     public ApiRequest HandleJsonResponse<T>(Action<T> responsePayload)
     {
-        this.responsePayload = (p) => JsonConvert.DeserializeObject<T>(p);
+        this.responsePayload = (p) => responsePayload.Invoke(JsonConvert.DeserializeObject<T>(p));
```

Comment:
Was this missing?

Reply:
Yes, that might actually solve the networking bug....thanks!